src/main/java/com/walking/intensive/chapter4/task16/Task16.java

```diff
@@ -41,7 +40,15 @@ static boolean isEqualSize(int[] arr1, int[] arr2) {
      * <p>Идентичными считаются массивы одинаковой длины, для которых arr1[i] == arr2[i] для любого i.
      */
     static boolean isEquals(int[] arr1, int[] arr2) {
-        // Ваш код
+        if (arr1.length == arr2.length) {
```

Comment:
if (arr1.length != arr2.length) {
           return false;
         }

         for (int i = 0; i < arr1.length; i++) {
              if (arr1[i] != arr2[i]) {
                  return false;
              }
          }

          return true;